frame/contracts/src/lib.rs

```diff
@@ -134,6 +134,9 @@ type BalanceOf<T> =
 type NegativeImbalanceOf<T> =
 	<<T as Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::NegativeImbalance;
 
+/// The current storage version.
+pub const STORAGE_VERSION: StorageVersion = StorageVersion::new(4);
```

Comment:
Why do we need this public const? Users can determine the version by using `trait GetStorageVersion`, right? This introduces a possibility for a mismatch: The pallet programmer uses a different value in the annotation.